samples/typescript_nodejs/00.empty-bot/src/index.ts

```diff
@@ -8,7 +8,8 @@ import * as restify from 'restify';
 import {
     CloudAdapter,
     ConfigurationServiceClientCredentialFactory,
```

Comment:
Looks like `ConfigurationServiceClientCredentialFactory` no longer used in these files and can be removed as well